tests/test_errors.py

```diff
@@ -21,7 +21,7 @@ def test_episode_already_exists_raise_exception(self):
         existing_path = os.path.join(self.files, filename)
         with open(existing_path, 'w'):
             with raises(PathExistsException):
-                self.tv.rename(fn, existing_path)
+                self.tv.rename(fn, existing_path, symlink=False)
```

Comment:
Rather than update this test can you make the `symlink` arg default to False in the rename method please.  That will better signify it's an optional parameter to the method.